cypress/utils/utils.ts

```diff
@@ -155,11 +155,11 @@ export function selectFilter(filterName: string, identifiedRisk?: boolean): void
             .click({ force: true });
     } else {
         cy.get("div.pf-c-input-group")
-            .eq(0)
-            .find(commonView.filterToggleButton)
+            .get(commonView.filterToggleButton)
```

Comment:
Please get a unique locator for filterToggleButton . for now this is okay .

Reply:
Thanks will ask for unique locator here.